File: Random/4.Pascals Triangle.py
def solve(numRows):
    x=[[1],[1,1]]
    if numRows==1:
        return [x[0]]
    elif numRows==2:
        return x
    else:
        for i in range(numRows-2):
            f=x[-1]
            length=len(x[-1])
            newx=[1]
            for i in range(1,length):
                newx.append(f[i-1]+f[i])
            newx.append(1)
            x.append(newx)
    return x
print(solve(2))

File: Random/4.Pascals Triangle.js
let solve = function (numsRows) {
  x = [[1], [1, 1]];

  if (numsRows === 1) return[x[0]];
  else if (numsRows === 2) return x;
  else {
    for (let i = 0; i < numsRows - 2; i++) {
      f = x.at(-1);
      newx = [1];
      for (let j = 1; j < f.length; j++) {
        newx.push(f[j - 1] + f[j]);
      }
      newx.push(1);
      x.push(newx);
    }
  }
  return x;
};
console.log(solve(5));
